pkg/kubelet/dockertools/manager.go

```diff
@@ -504,6 +504,10 @@ func (dm *DockerManager) runContainer(pod *api.Pod, container *api.Container, op
 	if len(containerHostname) > hostnameMaxLen {
 		containerHostname = containerHostname[:hostnameMaxLen]
 	}
+	namespacedName := types.NamespacedName{pod.Namespace, pod.Name}
+	labels := map[string]string{
+		"kubernetes.io/pod": namespacedName.String(),
```

Comment:
Docker's proposed namespace convention is reverse DNS IIRC: io.kubernetes.pod.name